mobile-app/.env.prod.dist

```diff
@@ -0,0 +1 @@
+HEX_API_URL=http://myhostname/api
```

Comment:
Where will this file be used? It's ok for the env one(in the docker-compose.dev) but I'm not sure for this one

Reply:
When we'll package our app as and APK for instance, we well need a `.env` file containing the URL for the API to use in production. This is where this template comes at play, it gives the template for this `.env` file.

But since our API is supposed to be open and public, I guess I could have filled it with the true URL right away... What do you think?

Reply:
I don't know how apk generation works but for the moment this file is not used anywhere so for me there is no need to keep it !

Reply:
I've updated its content to

```
HEX_API_URL=http://ec2-54-86-88-53.compute-1.amazonaws.com/api
```

For me it's a good thing to have both a local development config example, and a production environment example. What's more, if (one day) we get to automatize build of our mobile app, then in the script we may write something like `cp .env.prod.dist .env` to have this file automatically created.

Give me a call if you still disagree, so that we can resolve this together 🙂